libmscore/chord.cpp

```diff
@@ -732,6 +733,7 @@ void Chord::addLedgerLines()
 
                   if (note->visible())          // if one note is visible,
                         visible = true;         // all lines between it and the staff are visible
+                  hw = qMax(hw, note->headWidth());
```

Comment:
But `hw` is used in the same loop (see line 760/762 below), couldn't that cause any issues? Maybe it would be more reliable to find a maximal notehead width before executing this loop.

Reply:
Or is the intention to keep the ledger line small for small notes that lay further from a staff?

Reply:
Exactly - after reading through the different comments and looking at the different example, this seemed the most desirable solution.  So, if the outmost notes in a chord are small, they get small ledger lines.  As soon as a full-size note is encountered, the rest of the ledger lines from there to the staff are full-size as well.

It's not obvious unless you look closely, but check out the first and third chords in the image I showed in the issue thread:

![ledger-line-size](https://user-images.githubusercontent.com/1799936/59854376-0c902f80-9330-11e9-9599-97e82d89d760.png)

Reply:
OK, looks reasonable indeed, thanks!